.devfile.yaml

```diff
@@ -9,16 +9,29 @@
 schemaVersion: 2.1.0
 metadata:
   name: che
+  language: typescript
 attributes:
-  controller.devfile.io/storage-type: ephemeral
+  controller.devfile.io/scc: container-build
+  controller.devfile.io/storage-type: per-workspace
+projects:
+  - name: che
+    git:
+      remotes:
+        origin: 'https://github.com/ScrewTSW/che.git'
```

Comment:
personal repo

Reply:
yes, for testing purposes until draft is removed and to correctly pull the updated devfile.

Reply:
removed.